tests/foreman/ui/test_contentview.py

```diff
@@ -4297,3 +4298,28 @@ def test_positive_greedy_dep_solving_with_multiversion_packages(session, module_
             'name = {}'.format(property_name),
             conserve_param_value
         )
+
+
+@tier2
+def test_positive_create_container(session, module_org):
+    """Create container and register to satellite
+
+    id: 39842da3-94b4-4348-b7d9-94282ae4aaf8
+
+    setup:
+        1. Spin up container with unique name.
+        2. Install pre-built bootstrap RPM.
+        3. Register to Satellite
+        4. Search for content host to make sure it exists
+
+    expectedresults:  Content host successfully registered and
+        seen on list.
+
+    CaseLevel: Integration
+
+    CaseImportance: Critical
+    """
+    with ContainerContext() as chost:
+        chost.register(org=module_org.name)
+        with session:
+            assert session.contenthost.search(chost.name)[0]['Name'] == chost.name
```

Comment:
Do you have to do it from UI? It would be faster from API or CLI, if that is possible.

Also, this kind of looks like test verifying `robottelo.containers` implementation, not Satellite. Can you take a look at `tests/robottelo/test_vm.py` and see if something similar could be done for containers?

Reply:
@mirekdlugosz sure thing.  I'll check it out.